infrastructure/main.tf

```diff
@@ -183,6 +188,11 @@ module "claim-store-api" {
     STAFF_NOTIFICATIONS_SENDER = "noreply@reform.hmcts.net"
     STAFF_NOTIFICATIONS_RECIPIENT = "${data.azurerm_key_vault_secret.staff_email.value}"
 
+    // MILO
+    MILO_CSV_SENDER = "noreply@reform.hmcts.net"
+    MILO_CSV_RECIPIENT = "${data.azurerm_key_vault_secret.milo_recipient.value}"
+    MILO_CSV_SCHEDULE = "0 0 6 * * ?"
```

Comment:
Change this to `"-"` so that it's disabled until the CCD implementation is ready